sklearn/metrics/_ranking.py

```diff
@@ -23,6 +23,7 @@
 from functools import partial
 from numbers import Real
 
+import numpy
```

Comment:
We don't need this import since we use just below `import numpy as np` and use `np` in general.